subo/command/build.go

```diff
@@ -51,6 +51,14 @@ func BuildCmd() *cobra.Command {
 			shouldDockerBuild, _ := cmd.Flags().GetBool("docker")
 
 			useNative, _ := cmd.Flags().GetBool("native")
+			makeTarget, _ := cmd.Flags().GetString("make")
+
+			util.LogStart(fmt.Sprintf("make %s", makeTarget))
```

Comment:
let's only run  this if `makeTarget` is not an empty string. Otherwise, looks good and I tested it!

Reply:
(the whole block, i.e. only attempt to run make if the `makeTarget` isn't empty.